### PR TITLE
Cleaning pads interval (wash interval)

### PR DESCRIPTION
**Pull request by NEVdataDyne**
Adds the option to set the interval between two washing of the clean pads.

Replaces #460

### CODE REVIEW

**Comment by NEVdataDyne** [comment]
I still encounter an issue that I can't solve and need the help from someone who knows how to correct it, I don't know if the problem is in this code or the client (see https://github.com/DeebotUniverse/client.py/pull/376 )

File: custom_components/deebot/manifest.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "deebot_client"
   ],
   "requirements": [
-    "deebot-client==4.0.0",
+    "git+https://github.com/NEVdataDyne/client.py.git@dev#deebot-client==4.0.0dev0",
```
Comment (edenhaus) [inline custom_components/deebot/manifest.json]: Please revert this line

**Review by edenhaus — CHANGES_REQUESTED** [review]

**Comment by NEVdataDyne** [comment]
Tested successfully in HA

**Comment by edenhaus** [comment]
As this component was migrated directly to HA core, please open the PR in https://github.com/home-assistant/core. Please note that the component is called `ecovacs` in the core repo

I will close it here as no features will be accepted for the custom component.

**Comment by NEVdataDyne** [comment]
@edenhaus : I made this development by copying what had been done for the cleaning mode option and with your help and MVladislav's help. I have no idea what I need to do to replicate this in HA core.

Could you please advise what I should do?

**Comment by edenhaus** [comment]
First, wait until the client PR is merged, and then you only need to apply the Number change to the core repo

**Pull request closed without merging**